### DISCUSSION TITLE
ResourcePack upload

### DISCUSSION THREAD

**Discussion by xMikux** [Kira-NT/mc-publish#28] [Q&A] [answered]
Hi, I am trying to setup with this action to my workflows, but when I publish, the action fail with ``Error: At least one mod loader should be specified``

I didn't see any option for resourcepack.

**Comment by Kira-NT** [comment, marked as answer]
You need to specify at least one loader, because this is a requirement of the target platforms you upload your files to *(i.e., Modrinth anc CurseForge)*. 

Based on the response of the ["tag/loader"](https://api.modrinth.com/v2/tag/loader) endpoint, I can infer that Modrinth uses `minecraft` as a loader type for resource packs:
```json
{
  "icon": "...",
  "name": "minecraft",
  "supported_project_types": ["resourcepack"]
}
```

Therefore, you should be fine if you add this line to your configuration:
```yaml
  loaders: minecraft
```

Let me know how it goes :)

P.S. - don't forget about `game-versions` too:

```yaml
  game-versions: |
    1.18
    1.18.1
    1.18.2
    1.19
    1.19.1
    1.19.2
```

**Comment by xMikux** [comment]
Yeah, it work now, thanks for your help!

**Comment by xMikux** [comment]
It will be great to have some info on ``README.md`` file! Thanks

**Comment by Kira-NT** [comment]
I'll add it after I implement built-in support for resource packs :)

**Reply by meza** [reply]
Did you ever do this?

**Reply by Kira-NT** [reply]
Support for resource packs, plugins, and additional platforms is a part of the upcoming `v4`.
